.github/workflows/local_tests.yml

```diff
@@ -37,33 +37,33 @@ jobs:
             os: 'ubuntu-20.04'
             timeout-minutes: 80
 
-          - quarkus-version: '2.16.9.Final'
+          - quarkus-version: '2.16.11.Final'
             mandrel-version: '22.3.3.1-Final'
             java-version: '17'
             os: 'windows-2019'
             timeout-minutes: 80
-          - quarkus-version: '2.16.9.Final'
+          - quarkus-version: '2.16.11.Final'
             mandrel-version: '22.3.3.1-Final'
             java-version: '17'
             os: 'ubuntu-20.04'
             timeout-minutes: 80
 
-          - quarkus-version: '3.2.3.Final'
+          - quarkus-version: '3.2.6.Final'
             mandrel-version: '23.0.1.2-Final'
             java-version: '20'
             os: 'windows-2022'
             timeout-minutes: 80
-          - quarkus-version: '3.2.3.Final'
+          - quarkus-version: '3.2.6.Final'
             mandrel-version: '23.0.1.2-Final'
             java-version: '20'
             os: 'ubuntu-20.04'
             timeout-minutes: 130
```

Comment:
Does it still make sense to keep JDK 20 tests around?

Reply:
With JDK 21 released I'd say no.

Reply:
We don't have a JDK 21 release, so let's keep them there until we do :rofl: